tests/myria3d/test_train_and_predict.py

```diff
@@ -217,11 +216,13 @@ def _run_test_right_after_training(
     tmp_paths_overrides = _make_list_of_necesary_hydra_overrides_with_tmp_paths(
         toy_dataset_hdf5_path, tmpdir
     )
+    devices = "[0]" if accelerator == "gpu" else 1
```

Comment:
why enforcing it now? it looks like the previous versions could let you choose your gpu(s)

Reply:
In previous versions GPU 0 was always used. This is just a small refactor due to a change of signature of trainer, which requires two params: `accelerator` and `devices`, and deprecates gpus.
I made the choice to always use the same gpu to be able to run tests locally with the possibility to willingly avoid conflict with other processes (on other gpus, not on gpu 0). But I just discovered that lightning can do that automatically: 
https://lightning.ai/docs/pytorch/stable/accelerators/gpu_basic.html#find-usable-cuda-devices
So i'll update the test.